llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -1347,6 +1347,10 @@ RISCVTargetLowering::RISCVTargetLowering(const TargetMachine &TM,
         setOperationAction({ISD::STRICT_FP_ROUND, ISD::STRICT_FP_EXTEND}, VT,
                            Custom);
 
+        setOperationAction({ISD::EXPERIMENTAL_VP_STRIDED_LOAD,
```

Comment:
There's a FIXME on line 1342 that should be changed.

Reply:
Yeah, I hoping to be able to add lowering for the rest of the nodes mentioned there as well too. Will update

Reply:
I went ahead and merged the setOperationActions and updated the FIXME in 8d023b7d660dc5347740cb5e17e1dfbf8ff93686

Reply:
Thanks